Source/Readers/ImageReader/ImageDataDeserializer.cpp

```diff
@@ -73,10 +65,25 @@ IDataDeserializerPtr createImageDataDeserializer(const ConfigParameters& config,
     return deserializer;
 }
 
-IDataDeserializerPtr createImageDataDeserializer(const ConfigParameters& config)
+}
+
+std::unique_ptr<IDataDeserializer> createImageDataDeserializer(CorpusDescriptorPtr corpus, const ConfigParameters& config)
+{
+    auto dbg = std::string(config(L"labelType", "classification")); UNUSED(dbg);
+    auto dbg2 = std::string(config(L"precision", "float")); UNUSED(dbg2);
+    const LabelType labelType = AreEqualIgnoreCase(std::string(config(L"labelType", "classification")), "classification") ? LabelType::Classification : LabelType::Regression;
+    const ElementType elementType = AreEqualIgnoreCase(std::string(config(L"precision", "float")), "float") ? ElementType::tfloat : ElementType::tdouble;
+    return createImageDataDeserializer(corpus, config, labelType, elementType);
+}
+
+// (soon to be) deprecated version
+std::unique_ptr<IDataDeserializer> createImageDataDeserializer(const ConfigParameters& config)
 {
-    ImageConfigHelper configHelper(config);
-    return createImageDataDeserializer(config, configHelper);
+    auto dbg = std::string(config(L"labelType", "classification")); UNUSED(dbg);
+    auto dbg2 = std::string(config(L"precision", "float")); UNUSED(dbg2);
```

Comment:
Good catch. Seems like a left-over of my debugging code. They can both be removed.